letter.go

```diff
@@ -0,0 +1,118 @@
+	DeliveryMethod        string               `json:"delivery_method"`        //: "printed",
+	DirectMessageTo       string               `json:"direct_message_to"`      //: "Test",
+	DisplayTo             string               `json:"display_to"`             //: "Test Name",
+	DocumentDate          time.Time            `json:"document_date"`          //: "2021-05-11T21: 16: 23Z",
+	EmailTo               string               `json:"email_to"`               //: "test@email.me",
+	FailureUnacknowledged string               `json:"failure_unacknowledged"` //: false,
+	FaxStatus             string               `json:"fax_status"`             //: "success",
+	FaxAttachments        bool                 `json:"fax_attachments"`        //: true,
+	FaxTo                 string               `json:"fax_to"`                 //: "5555555555",
+	IsProcessed           bool                 `json:"is_processed"`           //: true,
+	LetterType            string               `json:"letter_type"`            //: "provider",
+	Patient               int64                `json:"patient"`                //: 140754680086529,
+	Practice              int64                `json:"practice"`               //: 140754674450436
+	SendToContact         *LetterSendToContact `json:"send_to_contact"`        //: {},
+	SendToElationUser     int64                `json:"send_to_elation_user"`   //: 1234,
+	SendToName            string               `json:"send_to_name"`           //: "Test Name",
+	SignDate              *time.Time           `json:"sign_date"`              //: "2021-05-11T21: 16: 23Z",
+	SignedBy              int64                `json:"signed_by"`              //: 12323455,
+	Subject               string               `json:"subject"`                //: "Test Subject",
+	Tags                  []any                `json:"tags"`                   //: [],
+	ToNumber              string               `json:"to_number"`              //: "5555555555",
+	ViewedAt              *time.Time           `json:"viewed_at"`              //: "2021-05-11T21: 16: 23Z",
+	WithArchive           bool                 `json:"with_archive"`           //: false,
+}
+
+type LetterAttachment struct {
+	ID           int64  `json:"id"`
+	DocumentType string `json:"document_type"`
+}
+
+type LetterSendToContact struct {
+	ID          int64                           `json:"id"`
+	FirstName   string                          `json:"first_name"`
+	LastName    string                          `json:"last_name"`
+	NPI         string                          `json:"npi"`
+	OrgName     string                          `json:"org_name"`
+	Specialties []LetterSendToContactSpeciality `json:"specialties"`
+}
+
+type LetterSendToContactSpeciality struct {
+	ID   int64  `json:"id"`
+	Name string `json:"name"`
+}
+
+type FindLettersOptions struct {
+	*Pagination
+
+	DocumentDateGT  time.Time `url:"document_date_gt,omitempty"`
```

Comment:
For all the gt/lt values - is it `document_date_gt` or `document_date__gt`? The docs seem to have the latter

Reply:
Missed the extra underscore. Will fix.